RallyHereIntegration/Source/RallyHereEditor/RallyHereEditor.Build.cs

```diff
@@ -4,6 +4,7 @@ public class RallyHereEditor : ModuleRules
 {
 	public RallyHereEditor(ReadOnlyTargetRules Target) : base(Target)
 	{
+		bEnforceIWYU = false;
```

Comment:
We want this to be defaulted (which is = true for engine plugins).

Reply:
We also want the default for the TreatAsEngineModule.

Likely can remove the two later flags (bEnableUndefinedIdentifierWarnings and bAllowConfidentialPlatforms)?